contracts/StMATIC.sol

```diff
@@ -373,7 +373,12 @@ contract StMATIC is
             .getOperatorInfos(true);
 
         for (uint256 i = 0; i < operatorInfos.length; i++) {
-            IValidatorShare(operatorInfos[i].validatorShare).withdrawRewards();
+            // Bug: Major 1.3
+            IValidatorShare vs = IValidatorShare(operatorInfos[i].validatorShare);
```

Comment:
This is good, but I've already fixed this one. 😸 
https://github.com/Shard-Labs/PoLido/blob/fix/post-audit/contracts/StMATIC.sol#L379